db/sensor.py

```diff
@@ -16,10 +16,18 @@
 from datetime import datetime
 
 from sqlalchemy import String, Integer, DateTime
+from sqlalchemy.ext.associationproxy import AssociationProxy, association_proxy
 from sqlalchemy.orm import relationship, mapped_column, Mapped
 
 from db.base import Base, AutoBaseMixin, ReleaseMixin
 
+from typing import List, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from .deployment import Deployment
+    from .thing import Thing
+    from .observation import Observation
```

Comment:
relative imports may fail when running this as an app from the root. to prevent this from happening you can change these to

`from db.deployment import Deployment`
`from db.thing import Thing`
`from db.observation import Observation`